tests/tx/test_verification.py

```diff
@@ -493,9 +563,216 @@ def test_merge_mined_block_validate_full(self) -> None:
         # MergeMinedBlock methods
         verify_aux_pow_wrapped.assert_called_once()
 
+    def test_poa_block_verify_basic(self) -> None:
```

Comment:
Should we move all PoA-related tests to the `tests/poa/` folder?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1064/commits/0c60081361add942bf9c8f8fd5c67a3475a43baf

Reply:
Thanks!